pyccel/codegen/printing/fcode.py

```diff
@@ -2653,16 +2653,7 @@ def _print_Header(self, expr):
         return ''
 
     def _print_ConstructorCall(self, expr):
-        func = expr.func
-        name = str(func.name)
-        if name == "__init__":
-            name = "create"
-        name = self._print(name)
-
-        code_args = ''
-        if expr.arguments is not None:
-            code_args = ', '.join(self._print(i) for i in expr.arguments)
-        code = '{0}({1})'.format(name, code_args)
+        code = self._print_FunctionCall(expr)
         return self._get_statement(code)
```

Comment:
Is this function still needed? Or can the print be done entirely through `_print_FunctionCall`?

Reply:
Fixed